yamux/src/session.rs

```diff
@@ -273,7 +297,11 @@ where
             }
         };
         let (frame_sender, frame_receiver) = channel(8);
-        self.streams.entry(stream_id).or_insert(frame_sender);
+
+        match self.streams.entry(stream_id) {
```

Comment:
Is it possible to reproduce the error branch in test?

Reply:
I will try to write the corresponding test in the future pr

Reply:
#252